src/lexer.coffee

```diff
@@ -192,13 +192,17 @@ exports.Lexer = class Lexer
     # what CoffeeScript would normally interpret as calls to functions named
     # `get` or `set`, i.e. `get({foo: function () {}})`.
     else if tag is 'PROPERTY' and prev
-      if prev.spaced and prev[0] in CALLABLE and /^[gs]et$/.test(prev[1]) and @tokens[@tokens.length - 2][0] isnt '.'
-        @error "'#{prev[1]}' cannot be used as a keyword, or as a function call without parentheses", prev[2]
+      if prev.spaced and prev[0] in CALLABLE and /^[gs]et$/.test(prev[1]) and
+         @tokens[@tokens.length - 2][0] not in ['.', '?.', '@']
```

Comment:
We're relying on `@tokens[@tokens.length - 2]` existing here / `- 3` in the else, but it seems it's fine.

Reply:
This is a good catch. If you file begins with `get foo: ->`, the compiler crashes. Updated. The compiler won’t throw the error if your file begins with `get` or `set`, but it will compile it as a function call; and it could only have been intended as a function call, as getters and setters need to be inside objects or classes.